docs/source/models/causes/neonatal/preterm_birth.rst

```diff
@@ -1,47 +1,221 @@
+
 Scope
 +++++
 
+The Level 4 neonatal conditions included in the MNCNH Portfolio model are closely linked via the 
+:ref:`Overall Neonatal Disorders Model <2021_cause_neonatal_disorders_mncnh>`.  Since the LBWSG Risk Factor has a PAF-of-one relationship with this Preterm Birth cause, we will need to ensure that the risk-stratified cause-specific mortality rates are zero for non-preterm categories, and their weighted average matches the overall cause-specific mortality rate.  This will produce internally consistent mortality risk values for all neonatal subcauses and LBWSG exposure levels. These risks can then be further individualized based on treatment coverage and efficacy, to ensure that individuals who have access to a specific existing or hypothetical treatment have a lower risk of mortality.
+
 Assumptions and Limitations
 +++++++++++++++++++++++++++
 
-Cause Model Diagram
-+++++++++++++++++++
+Focusing solely on YLLs will likely underestimate the total burden of this cause and, consequently, the DALYs averted by interventions that reduce prevalence and mortality due to it. However, we anticipate this underestimation to be less than 10%.
+
+GBD does not provide an estimate of the proportion of preterm births that are associated with RDS. We will need to make an assumption about this proportion based on data we obtain from another source.
+
+We might need to include the prevalence of Preterm Birth Complications, in order to quantify the cost and impact of corticosteroids intervention, which prevents RDS incidence (as opposed to the CPAP intervention, which reduces on mortality among neonates who have RDS).
+
+Cause Model Decision Graph
+++++++++++++++++++++++++++
+
+We are not modeling Preterm Birth dynamically as a finite state machine, but we can draw an directed 
+graph to represent the collapsed decision tree  
+representing this cause. Unlike a state machine representation, the values on the 
+transition arrows represent decision probabilities rather than rates per 
+unit time.
+
+
+.. graphviz::
+
+    digraph NN_preterm_decisions {
+        rankdir = LR;
+        lb [label="live birth", style=dashed]
+        nn_alive [label="neonate did not\ndie of preterm"]
+        nn_dead_RDS [label="neonate died\nof preterm with RDS"]
+        nn_dead_no_RDS [label="neonate died\nof preterm without RDS"]
+
+        lb -> nn_alive  [label = "1 - mr_w - mr_wo"]
+        lb -> nn_dead_RDS [label = "mr_w"]
+        lb -> nn_dead_no_RDS [label = "mr_wo"]
+    }
+
+.. list-table:: State Definitions
+    :widths: 7 20
+    :header-rows: 1
+
+    * - State
+      - Definition
+    * - live birth
+      - The parent simulant has given birth to a live child simulant (which
+        is determined in the
+        intrapartum step of the :ref:`pregnancy model
+        <other_models_pregnancy_closed_cohort_mncnh>`)
+    * - neonate survived
+      - The child simulant survived for the first 28 days of life
```

Comment:
The node name should be changed to "neonate did not die of preterm", and the definition should be changed accordingly.